AlexNet.js

```diff
@@ -195,12 +195,12 @@ function AlexNet() {
             if (showConvDims && index < architecture.length - 1) {
 
                 // Conv Dims
-                sprite = makeTextSprite(layer['filterWidth'].toString());
-                sprite.position.copy( conv_object.position ).sub( new THREE.Vector3( convFn(layer['filterWidth'])/2, -3, depthFn(layer['depth'])/2 ) );
+                sprite = makeTextSprite(layer['filterHeight'].toString());
+                sprite.position.copy( conv_object.position ).sub( new THREE.Vector3( convFn(layer['filterHeight'])/2, -3, depthFn(layer['depth'])/2 ) );
                 sprites.add( sprite );
 
-                sprite = makeTextSprite(layer['filterHeight'].toString());
-                sprite.position.copy( conv_object.position ).sub( new THREE.Vector3( -1, convFn(layer['filterHeight'])/2, depthFn(layer['depth'])/2 ) );
+                sprite = makeTextSprite(layer['filterWidth'].toString());
+                sprite.position.copy( conv_object.position ).sub( new THREE.Vector3( -1, convFn(layer['filterWidth'])/2, depthFn(layer['depth'])/2 ) );
```

Comment:
As same reason as above, this line should be changed like this.
```suggestion
                sprite.position.copy( conv_object.position ).sub( new THREE.Vector3( -1, convFn(layer['filterHeight'])/2, depthFn(layer['depth'])/2 ) );
```